GiEnJul/Controllers/RecipientController.cs

```diff
@@ -26,6 +26,11 @@ public RecipientController(IRecipientRepository recipientRepository, IPersonRepo
         public async Task<ActionResult<Entities.Recipient>> PostAsync([FromBody] Models.Recipient recipient)
         {
             _log.Debug("Adding recipient object: {@recipient}", recipient);
+            if (string.IsNullOrEmpty(recipient.Location)) {
```

Comment:
`if (string.IsNullOrEmpty(recipient.Location))`

This behaviour could be handled using Annotations in the Models.Recipient Class. 
Example
`[Required]
public string Location { get; set; }
`

Using Annotations, the method will return an error if Location is not set.